src/components/state-details.js

```diff
@@ -731,6 +738,12 @@ export default function StateDetailsPage({ location, data }) {
                 <strong>{solarPanelsCountStr} Megawatts</strong> of solar.
               </p>
 
+              <p className="mt-6">
+                Since the average wind turbine produces 2.75 MW of capacity,{" "}
```

Comment:
Sorry for the wording tweaks, but I think it should be "Since the average wind turbine **provides** 2.76 MW of capacity". Capacity is a property of the turbine, not a thing it _creates_ so to speak. It'd be like saying the average solar farm produces 1,000 sq feet of solar.